schema/products/rlie-s2.json

```diff
@@ -0,0 +1,156 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://example.com/rlie-s2.json",
+  "title": "Rive and Lake Ice Extent - Sentinel-2 STAC Schema",
+  "description": "This schema only checks the existence but the schema of the properties",
+  "oneOf": [
+    {
+      "$comment": "This is the schema for Rive and Lake Ice Extent - Sentinel-2 Items",
+      "type": "object",
+      "required": [
+        "stac_version",
+        "stac_extensions",
+        "type",
+        "id",
+        "bbox",
+        "properties",
+        "collection",
+        "links",
+        "assets"
+      ],
+      "properties": {
+        "stac_version": { "const": "1.0.0" },
+        "stac_extensions": {
+          "const": [
+            "https://stac-extensions.github.io/projection/v1.1.0/schema.json"
+          ]
+        },
+        "type": { "const": "Feature" },
```

Comment:
In this case we have feature, and then collection altogether in the same schema. Is that ok?

Reply:
Yes, the collection schema is down there at the [line 173](https://github.com/eea/clms-stac/pull/13/files#diff-6155886be815023250665be62c564f746c7a16e7206394731b909b1d55fced46R173)